frontend/src/components/NotificationsBar.js

```diff
@@ -92,29 +82,10 @@ export class NotificationsBar extends Component {
         </TabPanel>
 
         <TabPanel>
```

Comment:
Makes more sense and technically simpler as a single item tied to the FP than a list